packages/web/src/types.ts

```diff
@@ -0,0 +1,190 @@
+export type components = string[];
+
+export type children = any;
+
+export type data = any[];
+
+export type dataFieldArray =
+	| string
+	| string[];
+
+export interface dataNumberBox {
+	label?: string;
+	start: number;
+	end: number;
+}
+
+export type date =
+	| string
+	| string[];
+
+export type dateObject = object;
+
+export interface dateRange {
```

Comment:
What is this being used for?

Reply:
Good catch. This has been mirrored from core.
I think it's unused we can clean it up from both places.